layers/bsdiff/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG PHP_VERSION
+ARG BREF_VERSION
+FROM bref/build-php-$PHP_VERSION:$BREF_VERSION AS ext
```

Comment:
Like @Nyholm said, big things have happened. This file needs updating the same way the others are.

Reply:
It was updated as what @Nyholm suggested. My concern is that I accidentally merged new changes instead of doing a rebase. I can submit a new PR if needed.

Reply:
I see. There are additional changes to be made.